app/javascript/controllers/notifications_controller.js

```diff
@@ -0,0 +1,37 @@
+import { Controller } from "@hotwired/stimulus"
+
+// Connects to data-controller="notifications"
+export default class extends Controller {
```

Comment:
When user type something in the search section we can handle the actions via stimulus from the text box

Reply:
Next PR we will try to use this